pyproject.toml

```diff
@@ -21,7 +21,7 @@ bugsnag = "^3.6"
 requests = "^2.22"
 slackclient = "^1"
 PyYaml = "^5.1"
-blossom-wrapper = { git = "https://github.com/GrafeasGroup/blossom-wrapper.git", branch = "master" }
+blossom-wrapper = { git = "https://github.com/GrafeasGroup/blossom-wrapper.git", branch = "master", rev = "e8ed76b" }
```

Comment:
I'd actually leave this at master -- we have the ability to deploy everything at once, so it would make the most sense to have this stay the same.

Reply:
My thought process was that this makes it harder to have multiple branches open while updating the wrapper and we have to always remember to update all the bots at once -- I think we are bound to forget one at some point.

But I can change it back if you don't like it

Reply:
My concern is that the existing workflow is:

- make a slight change to the wrapper
- deploy the bot that relies on it, have that bot install it
- if multiple bots need the changes at once, then it's just `bubbles deploy all`

I worry that if we set this to a specific hash, we'll forget about it and newer changes won't get merged in. If we're worried about forgetting, maybe it would be a good idea to just make `bubbles deploy` default to refreshing everything.

Reply:
But if you are worried that we forget about it, wouldn't that mean that with the current workflow that bot would just break, because it's not adapted to the new wrapper?

Reply:
I moved this to a separate issue, #244.